tests/validators/test_list.py

```diff
@@ -86,25 +181,10 @@ def test_list_json():
     ]
 
 
-@pytest.mark.parametrize(
-    'input_value,expected',
-    [
-        ([], []),
-        ([1, '2', b'3'], [1, '2', b'3']),
-        (frozenset([1, '2', b'3']), Err('Input should be a valid list [type=list_type,')),
-        ((), []),
-        ((1, '2', b'3'), [1, '2', b'3']),
-        (deque([1, '2', b'3']), [1, '2', b'3']),
-        ({1, '2', b'3'}, Err('Input should be a valid list [type=list_type,')),
-    ],
-)
+@pytest.mark.parametrize('input_value,expected', [([], []), ([1, '2', b'3'], [1, '2', b'3'])])
```

Comment:
This is covered by tests above since we no longer special case "any" item validators.